cli/test/test_scaffold.py

```diff
@@ -0,0 +1,29 @@
+from click.testing import CliRunner
+
+import popper.commands.cmd_scaffold as scaffold
+import popper.commands.cmd_run as run
+from test_common import PopperTest
+
+import unittest
+import os
+
+
+class TestScaffold(PopperTest):
+
+    def test_scaffold(self):
+
+        repo = self.mk_repo()
+        runner = CliRunner()
+        file_loc = repo.working_dir+'/wf.yml'
+
+        result = runner.invoke(scaffold.cli, ['-f', file_loc])
+
+        self.assertEqual(result.exit_code, 0)
+        self.assertTrue(os.path.isfile(file_loc))
+
+        with self.assertLogs('popper') as test_logger:
+
+            result = runner.invoke(run.cli, ['-f', file_loc])
+            self.assertEqual(result.exit_code, 0)
+
+        self.assertTrue(len(test_logger.output))
```

Comment:
this should be indented within the `with` statement, as the `test_logger` variable is only valid in this scope

Reply:
we can also inspect the content of the output to ensure that it ran as it should

Reply:
Can I inspect the number of steps it ran because the other output is very lengthy?

Reply:
yeah, that works fine. we only need to solve the pep8 issues that might arise

Reply:
Updated.